src/Docker.PowerShell/Objects/ContainerOperations.cs

```diff
@@ -73,18 +74,48 @@ internal static class ContainerOperations
                     HostConfig = hostConfiguration
                 });
         }
+        
+        internal static async Task<IList<ContainerListResponse>> GetContainersById(string id, DotNet.DockerClient dkrClient)
+        {
+            return (await dkrClient.Containers.ListContainersAsync(new ContainersListParameters
```

Comment:
You actually dont need to await these since there are no result dependencies. You can just return the task instead.